node_modules/col-analytics/lib/util.js

```diff
@@ -23,7 +23,64 @@
  * ENHANCEMENTS, OR MODIFICATIONS.
  */
 
+var _ = require('lodash');
+
 var CourseAPI = require('col-course');
+var EventTypes = require('./constants');
+
+/**
+ * Certain object metadata is static and available elsewhere in the SuiteC schema. For example,
+ * the 'asset_created_at' property. Other metadata is esoteric (e.g., 'whiteboard_element_scale_y')
+ * and of little value. The 'events' table is best used to track properties that change over time,
+ * like 'asset_views'.
+ *
+ * @return {Array}                  Event metadata properties to exclude from 'events' table
+ */
+var EVENT_METADATA_BLACKLIST = [
+  'asset_categories',
+  'asset_created_at',
+  'asset_description',
+  'asset_description_hashtag',
+  'asset_description_length',
+  'asset_dislikes',
+  'asset_image_url',
+  'asset_liked',
+  'asset_mime',
+  'asset_source',
+  'asset_thumbnail_url',
+  'asset_type',
+  'asset_url',
+  'asset_users',
+  'comment_body',
```

Comment:
@johncrossman Is there any value to including `comment_body` and `asset_description` if they might be edited over time, or do you think they're just too lengthy and noisy?

Reply:
@paulkerschen  I excluded them for reasons you mention but they do qualify as mutable. I'll remove from blacklist.